password-generator-manager/src/utils.js

```diff
@@ -43,17 +43,17 @@ export function generateRandomPassword(length) {
     specialCharacters[Math.floor(Math.random() * specialCharacters.length)];
   password += randomCharacter;
 
+  // Remove whitespace, if any
+  password = password.replace(/ +/g, "");
+
   // Generate the remaining characters
   const remainingLength = length - 3;
-  const characters = words.join("") + randomNumber + specialCharacters.join("");
+  const characters = words.join("").replace(/ +/g, "") + randomNumber + specialCharacters.join("");
```

Comment:
By doing this `.replace(/ +/g, "")` here , we are essentially repeating the same thing twice (lines 46-48 and 51). Can we think of a better solution or put this piece of code someplace where we use it just once and it works always?

Reply:
Hi @utk09, I've removed the second call to replace. Please let me know what you think!